pkg/shares/share_builder.go

```diff
@@ -32,18 +33,12 @@ func NewBuilder(ns appns.Namespace, shareVersion uint8, isFirstShare bool) *Buil
 	}
 }
 
-func (b *Builder) Init() (*Builder, error) {
+// Init initializes the share builder. It should be called right after NewBuilder.
+func (b *Builder) Init() error {
```

Comment:
apologies if I just forgot, but if this function should be called right after `NewBuilder` then why don't we just call it there?

Reply:
Resolved via https://github.com/celestiaorg/celestia-app/pull/2165/commits/cb86e485f193c363038fc598ded216c0f1863350